<launch>

	<arg name="agent_name" default="freicar_X" />
	<arg name="requires_chaperone" default="true" />
	<arg name="safety_radius" default="0.1" />
	<arg name="safety_localization_time" default="0.5" />

	<node name="$(arg agent_name)_agent" pkg="freicar_agent" type="freicar_agent_node" output="screen">
		<param name="global_frame"       value="world" />
		<param name="agent_name"         value="$(arg agent_name)" />
		<param name="agent_state_topic"  value="/$(arg agent_name)/agent_state" />
		<param name="cmd_topic"          value="/freicar_agent_commands" />
		<param name="track_service"      value="/freicar_chaperone_track_srv" />
		<param name="requires_chaperone" value="$(arg requires_chaperone)" />
		<param name="safety_radius"      value="$(arg safety_radius)" />
		<param name="safety_localization_time" value="$(arg safety_localization_time)" />
	</node>
</launch>
